scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -19,11 +18,18 @@ label {
     border-color: $brand-primary;
   }
 
-  &[disabled],
-  &[readonly],
-  fieldset[disabled] & {
```

Comment:
why did we remove this `fieldset` selector?

Reply:
The linter says these are excessive selectors.

Reply:
how does removing it fixes things? This selector 
```css
fieldset[disabled] .form-control
```
is now being removed because the linter is complaining? Aren't we breaking stuff?